timon/plugins/__init__.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+# #############################################################################
+# Copyright : (C) 2023 by MHComm. All rights reserved
+#
+# __author__ = "Quentin Laymajoux"
+# __email__ = "info@mhcomm.fr"
+#
+# Name       : timon.plugins
+"""
+Summary: This folder contains all classic plugins supported by timon.
+All plugins have to inherit from timon.plugins.base.TimonBasePlugin
+
+All timon plugins must have coroutines `start` and `stop`.
+
+To permits importing plugins correctly, all plugin files must have a
+`plugin_cls` variable that corespond to the plugin class
+
+If you want to enable a plugin and use it, you have to add correct lines in the
+plugins section of the timon config file with correct params + an extra
+`enabled` param set to True. The plugin name will correspond to the plugin file
+name.
```

Comment:
perhaps clarify or give an example. do you really have to specofy the realtive or absolute file name or do you mean module name?

Reply:
comment changed